.github/workflows/ramalama.yaml

```diff
@@ -0,0 +1,165 @@
+            curl -L "http://ftp.lysator.liu.se/pub/opensuse/repositories/devel:/kubic:/libcontainers:/unstable/xUbuntu_${ubuntu_version}/Release.key" | sudo apt-key add - && \
+            sudo apt-get update && \
+            sudo apt-get -y install podman; }
+          podman version
+
+      - name: Revert unprivileged user namespace restrictions in Ubuntu 24.04
+        run: |
+          # allow unprivileged user namespace
+          sudo sysctl -w kernel.apparmor_restrict_unprivileged_userns=0
+
+      - name: Set cgroup_manager to 'cgroupfs' instead of systemd
+        run: |
+          mkdir -p ~/.config/containers
+          cat <<EOT >> ~/.config/containers/containers.conf
+          [engine]
+          cgroup_manager="cgroupfs"
+          EOT
+          podman info
+
+      - name: Install pnpm deps and build Podman Desktop
+        working-directory: ./podman-desktop
+        run: |
+          pnpm install --frozen-lockfile
+          pnpm test:e2e:build
+
+      - name: Ensure getting current HEAD version of the test framework
+        working-directory: ./podman-desktop-extension-ai-lab/tests/playwright
+        run: |
+          # workaround for https://github.com/containers/podman-desktop-extension-bootc/issues/712
+          version=$(npm view @podman-desktop/tests-playwright@next version)
+          echo "Version of @podman-desktop/tests-playwright to be used: $version"
+          jq --arg version "$version" '.devDependencies."@podman-desktop/tests-playwright" = $version' package.json > package.json_tmp && mv package.json_tmp package.json
+
+      - name: Execute pnpm in AI Lab Extension
+        working-directory: ./podman-desktop-extension-ai-lab
+        run: pnpm install --no-frozen-lockfile
+
+      - name: Update ramalama image references in AI Lab Extension
+        working-directory: ./podman-desktop-extension-ai-lab
+        run: sed -i -E "s/(@sha256:[0-9a-f]+)/:${{ github.event.inputs.tag }}/g" packages/backend/src/assets/inference-images.json
+
+      - name: Build Image
+        working-directory: ./podman-desktop-extension-ai-lab
+        id: build-image
+        run: |
+          pnpm build
+          podman build -t local_ai_lab_image ./
+          CONTAINER_ID=$(podman create localhost/local_ai_lab_image --entrypoint "")
+          mkdir -p tests/playwright/tests/playwright/output/ai-lab-tests-pd/plugins
+          podman export $CONTAINER_ID | tar -x -C tests/playwright/tests/playwright/output/ai-lab-tests-pd/plugins/
+          podman rm -f $CONTAINER_ID
+          podman rmi -f localhost/local_ai_lab_image:latest
+
+      - name: Run E2E Smoke tests
+        working-directory: ./podman-desktop-extension-ai-lab
+        env:
+          PODMAN_DESKTOP_ARGS: ${{ github.workspace }}/podman-desktop
+          EXTENSION_PREINSTALLED: true
+        run: pnpm test:e2e:smoke
```

Comment:
It would be interesting to find out how stable is running all tests, not just smoke ones. But it might be follow up.